plugins/smart_proxy_remote_execution_ssh/remote_execution_ssh.rb

```diff
@@ -1 +1,2 @@
-gem 'smart_proxy_remote_execution_ssh', '0.1.3'
+gem 'smart_proxy_remote_execution_ssh', '0.1.4'
+gem 'foreman_remote_execution_core'
```

Comment:
Not really: the problem is with Ruby 1.8.7, which we need to support on proxy side for some distros ,while the core package is 1.9.3+. Therefore, we can't put it into deps of smart_proxy_remote_execution_ssh directly (on RHEL6, we delegate the execution to another process running in SCL)